packages/core/__tests__/Resource/Collection.test.js

```diff
@@ -19,7 +19,7 @@ describe('Collection tests', () => {
     find: repositoryFindMock,
   };
 
-  const collection = Collection('product', productRepository);
+  const collection = Collection.create('product', productRepository);
```

Comment:
het is dat het een test is, maar anders was een review comment geweest om het productCollection te noemen, dit ook omdat je verderop een newCollection, dan wordt het allemaal ietsjes duidelijker

Reply:
True, later ben ik ze ook wat specifieker gaan noemen.